strategy/to_json/from_object_containing.ts

```diff
@@ -1,17 +1,20 @@
 // Copyright 2018-2021 Gamebridge.ai authors. All rights reserved. MIT license.
 
 import { JSONObject, JSONValue, Serializable } from "../../serializable.ts";
+import { ToJSONStrategy } from "../compose_strategy.ts";
 
-/** convert `{ [_: string]: Serializable }` to `{ [_: string]: Serializable.toSerialize() }` */
-export function fromObjectContaining(
-  value: Record<string, Serializable | Serializable[]>,
-): JSONObject {
-  const record: Record<string, JSONValue> = {};
-  for (const prop in value) {
-    record[prop] = Array.isArray(value[prop])
-      ? (value[prop] as Serializable[]).map((v) => v.tsSerialize())
-      : (value[prop] as Serializable).tsSerialize();
-  }
+/** convert `{ [_: string]: Serializable }` to `{ [_: string]: Serializable.tsSerialize() }` */
+export function fromObjectContaining(): ToJSONStrategy {
+  return (
+    value: Record<string, Serializable | Serializable[]>,
+  ): JSONObject => {
+    const record: Record<string, JSONValue> = {};
+    for (const [prop, obj] of Object.entries(value)) {
```

Comment:
Nice yeah, we don't need to support IE11 anymore

![image](https://user-images.githubusercontent.com/17057295/141524590-04a8e125-d219-49fd-a5bc-068d96ad1fd3.png)

Reply:
Yea, it also allows the compiler to infer `obj`'s type, instead of us having to cast it with `value[prop]`. so its nice that way as well